<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InnerShopW</class>
 <widget class="QDialog" name="InnerShopW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>523</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupTop">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="4,0">
      <item row="0" column="1" rowspan="3" alignment="Qt::AlignRight">
       <widget class="QLabel" name="labPic">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,10">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>10</number>
        </property>
        <item row="0" column="1">
         <widget class="QLabel" name="labBook">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labRest">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labLabel">
          <property name="styleSheet">
           <string notr="true">color: rgb(148, 0, 0);
font-size: 16px;
font-weight:bold;</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labName">
        <property name="styleSheet">
         <string notr="true">font-size:18px;
font-weight:bold;
color:rgb(87,89,91);</string>
        </property>
        <property name="text">
         <string>秘制黄焖鸡米饭</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,10">
        <item row="0" column="1">
         <widget class="QLabel" name="labNotice">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font-weight:bold;</string>
          </property>
          <property name="text">
           <string>公告:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>点菜</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1" rowspan="2">
        <widget class="QListWidget" name="foodList">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>218</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="push1">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>套餐</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="push3">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>其它</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="push2">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>单品</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>评价</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_14">
       <item row="0" column="0">
        <widget class="QListWidget" name="commList">
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>商家</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_13" rowstretch="1,1,1,1,10,0">
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayout_9" columnstretch="5,1,40">
            <item row="0" column="1">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>配送服务:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resource.qrc">:/new/prefix1/res/配送.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="labWhoDe">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_8" columnstretch="1,10">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resource.qrc">:/new/prefix1/res/电话.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="labPhone">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="Line" name="line">
              <property name="styleSheet">
               <string notr="true">border:5px solid rgb(133,192,228);</string>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QGridLayout" name="gridLayout_10" columnstretch="13,1,100">
            <item row="0" column="1">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>配送时间:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="labTime">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resource.qrc">:/new/prefix1/res/时间.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="3">
             <widget class="Line" name="line_2">
              <property name="styleSheet">
               <string notr="true">border:5px solid rgb(133,192,228);</string>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_11" columnstretch="1,10">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resource.qrc">:/new/prefix1/res/地址.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="labAddress">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <layout class="QGridLayout" name="gridLayout_7" columnstretch="12,1,100">
            <item row="0" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resource.qrc">:/new/prefix1/res/公告.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="labNotice_2">
              <property name="text">
               <string>公告:</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>公告:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>110</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
